### PR TITLE
feat(post): add new post

### PR DESCRIPTION
**Pull request by arswysocki**
Blah-blah

### CODE REVIEW

**Comment by arswysocki** [comment]
I have added a new post for example

**Comment by profencer** [comment]
I would like you to prive better wording for post title

File: content/blog/otus-post-2/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Lesson Post 2!
```
Comment (profencer) [inline content/blog/otus-post-2/index.md]: This place to change

File: content/blog/otus-post-2/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+date: "2022-08-19T19:35:03.284Z"
+description: "Example on data management via VCS"
+---
+
```
Comment (arswysocki) [inline content/blog/otus-post-2/index.md]: Also would like to have another description

File: content/blog/otus-post-2/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+This post was created during the Otus lesson about Git and GitHub
```
Comment (arswysocki) [inline content/blog/otus-post-2/index.md]: Text needs rewording

**Pull request merged**